client/src/Main.elm

```diff
@@ -9,12 +9,12 @@ import View
 
 subscriptions : Model -> Sub Msg
 subscriptions model =
-    Time.every (Time.second * 2) UpdateMode
+    Time.every 2000 UpdateMode
```

Comment:
Time has changed a fair bit. It uses POSTIX time. 1000 is a second, so 2000 is 2 seconds, the same as before.